src/test/java/co/org/osso/api/ApiApplicationTests.java

```diff
@@ -1,16 +1,40 @@
 package co.org.osso.api;
 
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import java.util.List;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class ApiApplicationTests {
 
+	@Autowired
+	private BookController controller_target;
```

Comment:
We shouldn't use `_` for variable names in Java unless we're writing `private static final String MY_CONSTANT = "only OK for constant fields";`

Reply:
Done